test/e2e/kubectl/kubectl.go

```diff
@@ -861,7 +861,7 @@ metadata:
 			Testname: Kubectl, diff Deployment
```

Comment:
This went GA in 1.18 but is not added to the conformance suite until 1.19. This metadata actually represents when it was added to the suite. For example, someone can actually be conformant with 1.18 without meeting these tests.

Same as below. Other than that this is good to go.

Reply:
That's sort of bad, since we really want k8s provider to support that. Otherwise the `kubectl diff` feature can be broken for these cluster.

Reply:
tagging as conformance for 1.19 doesn't seem too problematic

Reply:
Ah retagged to 1.19